lib/assets.js

```diff
@@ -1,10 +1,10 @@
 'use strict';
 
-var merge = require('lodash').merge;
-var gulpif = require('gulp-if');
-var imagemin = require('gulp-imagemin');
+import {merge} from 'lodash';
+import gulpif from 'gulp-if';
+import imagemin from 'gulp-imagemin';
 
-var defaults = {
+const defaults = {
```

Comment:
To expand (as apparently @erikjung was arguing the inverse!), I don't think this is _incorrect_ so much as it implies that these are the *final, unchangeable* settings that this task will use when in fact they get munged in with args. So, it can be argued that the `defaults` object is constant, yes, technically, but I don't know that this is in the spirit of the `const` keyword?